app/templates/exercise.html

```diff
@@ -0,0 +1,20 @@
+{% extends "base.html" %}
+
+
+{% block content %}
+
+    <div class="container mt-4">
+        <div class="row">
+            <!-- Right side -->
+            <div class="col-5">
+                <!-- Exercise -->
+                {% if user.is_active_exercise %}
+                    <div class="exercise">
+                         <img src="{{ url_for('static', path=exercise) }}" alt="exercise image" style="width:270%;height:200%">
```

Comment:
Prefer width and height as attributes, not inside `style`

Reply:
fix the issue